### PR TITLE
Add TDP 37 & Update a bunch of Headers

### PR DESCRIPTION
**Pull request by mzero2nine**
Hi @dimsome 

I added TDP 37 and updated a bunch of outdated headers. Would be great if you could quickly review and merge, thanks!

### CODE REVIEW

File: MCCP/mccp-19.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ mccp: 19
 title: Disable automated collection of fees for Mainnet Save redemptions
 status: WIP
```
Comment (dimsome) [inline MCCP/mccp-19.md:4]: proposed?

**Review by dimsome — CHANGES_REQUESTED** [review]
just one smool change :)

File: MCCP/mccp-19.md (hunk 1 above)
Comment (mzero2nine) [reply MCCP/mccp-19.md]: Good catch, there's always one smol one isn't there? :D

Comment (dimsome) [reply MCCP/mccp-19.md]: cool thanks

**Review by dimsome — APPROVED** [review]

**Review by dimsome — APPROVED** [review]
lgtm

**Pull request merged**